src/ui/components/Rating/styles.scss

```diff
@@ -12,13 +15,51 @@
     grid-column-gap: 4px;
     min-height: 13px;
     width: min-content;
+
+    .IconStar {
+      height: $icon-medium;
+      width: $icon-small;
+    }
+
+    &.Rating--editable {
+      .IconStar {
+        width: $icon-medium;
+
+        // stylelint-disable max-nesting-depth
+        @include respond-to(large) {
+          width: $icon-small;
+        }
+      }
+    }
   }
 
   // The width of large rating stars are controlled by the container.
   &.Rating--large {
     grid-column-gap: 6px;
     min-height: 48px;
     width: 100%;
+
+    .IconStar {
+      max-height: $icon-large;
+      max-width: $icon-large;
+      width: 100%;
+
+      @media (orientation: portrait) {
+        padding: 0 6px;
```

Comment:
Can you give me a URL of a screen that this applies to? I don't see anything creating components like `<Rating styleSize="large">` (or `<UserRating...>`) but maybe I'm just missing it.

Reply:
this should happen on mobile and portrait:

![Alt text](https://monosnap.com/image/fYk5uXteQcsVYznsXhKAI4Oy1Xz0at.png)

I've slightly modified this tho to match it maybe a little bit better